lib/reply.js

```diff
@@ -138,7 +138,7 @@ Reply.prototype.send = function (payload) {
 
   if (this[kReplySerializer] !== null) {
     payload = this[kReplySerializer](payload)
-  } else if (hasContentType === false || contentType.indexOf('application/json') > -1) {
+  } else if (hasContentType === false || /^application\/([a-z.-]+\+)?json/.test(contentType)) {
```

Comment:
I do not think a regular expression is acceptable in this path. Additionally, this regular expression is not at all clear in intent. At a minimum, some comment to explain it needs to be added.

Reply:
Hi @jsumners, thanks for your return!
Regarding the comment, I agree. I can add it. I would like to know if the solution is acceptable to some extent.

Regarding the regular expression, is it about the performance? I can try an alternative with a list of content-types. I am just not sure how much better it will be in terms of performance.

In general, my idea is to accept any json content-type in the format "application/[anything here]/json)".

Reply:
@jsumners I added some tests and fixed the return type.

Reply:
I reduced it to `val.indexOf("json") > -1;` as suggested by @kibertoad.